torchvision/datasets/svhn.py

```diff
@@ -43,12 +43,7 @@ def __init__(self, root, split='train', transform=None, target_transform=None,
                  download=False):
         super(SVHN, self).__init__(root, transform=transform,
                                    target_transform=target_transform)
-        self.split = split  # training set or test set or extra set
-
-        if self.split not in self.split_list:
-            raise ValueError('Wrong split entered! Please use split="train" '
-                             'or split="extra" or split="test"')
-
+        self.split = verify_str_arg(split, ("train", "extra", "test"), "split")
```

Comment:
I think being explicit is good. But I wonder if it would make sense to just use `self.split_list`, as it was before?